### DISCUSSION TITLE
パッドはずしていまいました。

### DISCUSSION THREAD

**Discussion by takashiamimoto** [Yowkees/keyball#362] [Q&A] [answered]
ハンダ初心者です。

昔からあるハンダを使っていたのですが、あまりにもできなさすぎる感じでした。
色々調べていると、このハンダではダメと気づき慎重しました。
雲泥の差です。

パッド外れです。どこをジャンパーすればよろしいでしょうか。是非ともkeyball 使いたいのでよろしくお願いします。’
![IMG_3630](https://github.com/Yowkees/keyball/assets/56436832/59c668b8-b0f0-4729-bcd3-0a39c36771b1)

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

状況了解しました。
下記画像の同色どうしの端子をエナメル線もしくはゴム被覆導線で接続していただくと、復活できると思います。
なお、キーソケットの保持力が弱くなり、スイッチを挿したときにキーソケットが脱落してしまう可能性がありますので、
キーソケットを両面テープ等で基板に張り付けることをお勧めします。

以上、お手数ですがご対応お願いいたします。
![KB39padfix](https://github.com/Yowkees/keyball/assets/84209772/843ca96a-8e92-43b3-adf0-ded4646adf9a)

**Reply by takashiamimoto** [reply]
ありがとうございました。
早速使わして頂いています。

パソコン触る時とても姿勢が悪いので、いままで窮屈な姿勢でしたが、これからとても快適なパソコンライフを過ごさせていただきます。

**Comment by Yowkees** [comment]
無事にご使用いただけているようで安心しました。分割キーボードで私も肩こりがだいぶ楽になりました。
ぜひ末永くご愛用いただければ幸いです。
